File: src/test/java/integration/auth/code/RandomCodeServiceTest.java
package integration.auth.code;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.Test;

import com.matag.admin.auth.codes.RandomCodeService;

public class RandomCodeServiceTest {
  private RandomCodeService randomCodeService = new RandomCodeService();

  @Test
  public void generateRandomCode() {
    var code = randomCodeService.generatesRandomCode();

    assertThat(code).hasSize(10);
  }
}

File: src/test/java/integration/auth/code/RandomCodeServiceTest.kt
package integration.auth.code

import com.matag.admin.auth.codes.RandomCodeService
import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test

class RandomCodeServiceTest {
    private val randomCodeService = RandomCodeService()

    @Test
    fun generateRandomCode() {
        val code = randomCodeService.generatesRandomCode()

        Assertions.assertThat(code).hasSize(10)
    }
}
